src/app/code/community/Zendesk/Zendesk/Block/Adminhtml/Dashboard/Grids.php

```diff
@@ -65,15 +65,20 @@ protected function _prepareLayout() {
             } catch (Exception $ex) {
                 $allTicketView = array();
             }
-                
+
+            // Loop through each view ID as per config
             foreach($viewsIds as $viewId) {
-                $view = array_shift(array_filter($allTicketView, function($view) use($viewId) {
-                    return $view['id'] === (int) $viewId;
-                }));
-                
-                $count = array_shift(array_filter($ticketsCounts['view_counts'], function($view) use($viewId) {
+                // Searches for the view's details by matching all views retrieved from the api to the current view id
+                $view = array_filter($allTicketView, function($ticketView) use($viewId) {
+                    return $ticketView['id'] === (int) $viewId;
+                });
+                // Return only the first value (usually returns just 1)
+                $view = array_shift($view);
```

Comment:
If `$allTicketView` came as `array()` from the previous `try/catch`, or it's an empty array for whatever reason, you'll end up having `$view == NULL` here, because `array_shift(array())` returns `NULL`.

That wouldn't be much of a problem if the code on this file was checking it, but I don't think it's the case. For example I've seen `$view['title']` being used assuming that there is a valid value on `$view`.

Just pointing this out because if the reason of this PR is prevent a strict notice, I reckon we could still generate notices and unexpected results here under given circumstances.